src/clusterize.rs

```diff
@@ -130,7 +143,7 @@ pub fn build_meshlets_flex(
     split_factor: f32,
 ) -> Meshlets {
     let meshlet_count =
-        unsafe { ffi::meshopt_buildMeshletsBound(indices.len(), max_vertices, max_triangles) };
+        unsafe { ffi::meshopt_buildMeshletsBound(indices.len(), max_vertices, min_triangles) };
```

Comment:
Why did you change this? The 3rd's argument name of buildMeshletsBound seems to be max_triangles.